lib/token.js

```diff
@@ -127,6 +128,39 @@ function addFragmentListener(sdk, windowEl, timeout) {
   return deferred.promise.timeout(timeout || 120000, new AuthSdkError('OAuth flow timed out'));
 }
 
+function exchangeCodeForToken(sdk, oauthParams, authorizationCode, urls) {
+  // PKCE authorization_code flow
+  // Retrieve saved values and build oauthParams for call to /token
+  var meta = pkce.loadMeta(sdk);
+  var clonedParams = util.clone(oauthParams) || {};
+  util.extend(clonedParams, {
+    grantType: 'authorization_code',
+    authorizationCode: authorizationCode,
+    codeVerifier: meta.codeVerifier,
+    responseType: meta.responseType,
+    redirectUri: meta.redirectUri
+  });
+  delete clonedParams.state;
```

Comment:
As noted in an earlier comment, this is a **breaking change**. I suggest keeping the `state` value in the cookie as it is currently used by developers.

Reply:
These are the params passed to the /token endpoint. We have already hit /authorize and passed the state checks with the cookie. /token does not return to us state, so we cannot compare it. Otherwise I am cloning so as not to worry/think about what other parameters the user wants to pass to /token.

I'm confused how this is a breaking change since we don't currently hit the /token endpoint in this library.

Reply:
Apologies, my mistake. I was drawing a mapping of this object to a delete cookie call. This should be fine.